FVCalc.java

```diff
@@ -1,6 +1,23 @@
 // Computes the future value of a saving investment.
 public class FVCalc {
 	public static void main(String[] args){
-		// Replace this comment with your code
+		
+		int currentValue = Integer.parseInt(args[0]);		//the invested sum
+		double aiRate = Double.parseDouble(args[1]);			// the annual interest rate
```

Comment:
Indicative names.. I would say that seeing "ai", the first thing that comes to mind is Artificial Intelligence.. not Annual Interest.. 
double annualIntrestRate = Double.parseDouble(args[1]);